finality-aleph/src/validator_network.rs

```diff
@@ -0,0 +1,41 @@
+use aleph_primitives::AuthorityId;
+use futures::future::pending;
+
+/// Network represents an interface for opening and closing connections with other Validators,
+/// and sending direct messages between them.
+///
+/// Note on Network reliability and security: it is neither assumed that the sent messages must be
+/// always delivered, nor the established connections must be secure in any way. If the Network
+/// implementation fails to deliver a message, it may assume that the send method will be called
+/// again.
```

Comment:
Replace the last sentence with "The Network implementation might fail to deliver any specific message, so messages have to be resend while they still should be delivered."

Reply:
fixed